modules/core/src/test/java/org/apache/ignite/testframework/junits/GridAbstractTest.java

```diff
@@ -2661,6 +2664,23 @@ private void evaluateInsideFixture(Statement stmt) throws Throwable {
         }
     }
 
+    /**
+     * Clearing the static log for the class. <br/>
+     * There is a situation when class logs cannot be listened to although they
+     * are visible, for example, in a file. This happens when the test is in
+     * one of the suites and the static log was installed earlier and is not
+     * reset when the next test class is launched. To prevent this from
+     * happening, before starting all the tests in the test class, you need to
+     * reset the static class log.
+     *
+     * @param cls Class.
+     * */
+    protected void clearStaticLog(Class<?> cls) {
```

Comment:
Please add this method invocation to `GridAbstractTest#beforeTestsStarted` and `GridAbstractTest#afterTestsStopped` methods